monkey/infection_monkey/network/tcp_scanner.py

```diff
@@ -37,17 +37,22 @@ def _get_closed_port_data(port: int) -> PortScanData:
     return PortScanData(port, PortStatus.CLOSED, None, None)
 
 
-def _check_tcp_ports(ip: str, ports: List[int], timeout: float = DEFAULT_TIMEOUT):
+def _check_tcp_ports(
+    ip: str, ports: List[int], timeout: float = DEFAULT_TIMEOUT
+) -> Dict[str, List]:
     """
     Checks whether any of the given ports are open on a target IP.
     :param ip:  IP of host to attack
     :param ports: List of ports to attack. Must not be empty.
     :param timeout: Amount of time to wait for connection
-    :return: List of open ports.
+    :return: Dict with list of open ports and list of banners.
```

Comment:
This should be a dict where the keys are open ports and the values are banners. Thus the type hint should be `Dict[int, banner]`